buildSrc/src/main/kotlin/software/aws/toolkits/gradle/intellij/IdeVersions.kt

```diff
@@ -27,6 +27,9 @@ object IdeVersions {
                 "com.intellij.database",
                 "Pythonid:202.6397.98"
             ),
+            riderPlugins = listOf(
+                "rider-plugins-appender"
```

Comment:
Can we add a comment about what this does?

Reply:
Ah I see this used to be burred somewhere, still would be nice with a comment

Reply:
Ya we have an old comment somewhere, ill copy it over